src/runtime/Makefile

```diff
@@ -611,6 +611,9 @@ endif
 go-test: $(GENERATED_FILES)
 	go test -v -mod=vendor ./...
 
+fast-test: $(GENERATED_FILES)
+	for s in $$(go list ./...); do if ! go test -failfast -v -mod=vendor -p 1 $$s; then break; fi; done
```

Comment:
I wonder if it's worth putting this into a separate script (to make it clearer and avoid all the extra quoting). The script could then run the unit tests for each package in parallel (using `&`). It may also help make the "fast-fail" fail faster if the script is using `set -e` (assuming the failing test is not the last one to run ;)